src/patterns/exit-a-page-quickly/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Exit a page quickly
+description: Give users a way to quickly exit a service, website or application quickly and safely.
+section: Patterns
+theme: Help users to…
+aliases:
+backlog_issue_id: 213
+layout: layout-pane.njk
+status: Experimental
+statusMessage: This component is currently experimental. You'll need to do your own research to decide whether to use this pattern and add the <a class="govuk-link" href="/components/exit-this-page/">Exit this page</a> component to your service.
+---
+
+{% from "_example.njk" import example %}
+
+Give users a way to quickly exit a service, website or application quickly and safely.
+Use this pattern to add the [Exit this page](/components/exit-this-page/) component to your service and help users use it safely to protect their privacy.
+
+## When to use this pattern
+
+Use this pattern to help the user protect their privacy when your service contains sensitive information that could:
+- put someone at risk of abuse or retaliation
+- reveal someone’s plans to avoid or escape from harm
+
+For example, a potential victim using a service to help them leave domestic abuse.
+
+Other situations and topics where sensitive information could put users at risk might include:
+- rape or sexual assault
+- child abuse and neglect
+- any other type of abuse
+- stalking and harassment
+- reporting crime or fraud (whistleblowing)
+
+## When not to use this pattern
+
+You should not use this pattern for standalone content pages, such as dashboards and guidance. In these cases, use the Exit this page component on its own.
+
+## How it works
+
+First, you’ll need to add the [Exit this page](/components/exit-this-page/) component to your service and decide how it should work within your service.
+
+The component has several parts, including:
+- a button, that takes the user to another website
+- a secondary link, to give people that use assistive technologies a way to activate the component
+- a loading overlay, to immediately clears any content off the browser until the next website loads
+
+Next, you’ll need to create new content pages in your service.
+
+The purpose of these pages are to:
+- tell the users how to use ‘Exit this page’ to protect their privacy
+- give users information they need to stay safe online
```

Comment:
```suggestion
- give users information they need to stay safe online (this is called the safety content page)
```

Reply:
adding these couple of bits in brackets to make it clear how the following content sections relate to this intro/summary.

Reply:
I like these!